### PR TITLE
Docs: Relabel "Single version documentation" documentation from feature to explanation (Diátaxis)

### PR DESCRIPTION
**Pull request by benjaoming**
This ~is~ was a WIP.

Refs: https://github.com/readthedocs/readthedocs.org/issues/9746

---
:books: Documentation previews :books:

- User's documentation (`docs`): https://docs--9835.org.readthedocs.build/en/9835/

- Developer's documentation (`dev`): https://dev--9835.org.readthedocs.build/en/9835/

### CODE REVIEW

File: docs/user/single_version.rst [hunk 1]
```diff
@@ -1,18 +1,24 @@
-Single Version Documentation
+Single version documentation
 ----------------------------
 
-Single Version Documentation lets you serve your docs at a root domain.
+*Single version* documentation lets you serve your documentation direct from ``https://docs.example.com`` or ``https://example.readthedocs.io``.
```
Comment (ericholscher) [inline docs/user/single_version.rst:4]:
```suggestion
*Single version* documentation lets you serve your documentation at the top level of your domain,
for example ``https://docs.example.com/`` or ``https://example.readthedocs.io/``.
```

File: docs/user/single_version.rst [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
+
 By default, all documentation served by Read the Docs has a root of ``/<language>/<version>/``.
```
Comment (ericholscher) [inline docs/user/single_version.rst:6]:
```suggestion
By default, all documentation served by Read the Docs has a standard path of ``/<language>/<version>/``.
```

File: docs/user/single_version.rst [hunk 3]
```diff
@@ -1,18 +1,24 @@
...
-But, if you enable the "Single Version" option for a project, its documentation will instead be served at ``/``.
+But, if you enable the *single version* option for a project, its documentation will instead be served at ``/``.
```
Comment (ericholscher) [inline docs/user/single_version.rst:7]:
```suggestion
Documentation can be served at ``/`` if you enable the *single version* option for a project. 
```

Clearer I think.

File: docs/user/single_version.rst [hunk 4]
```diff
@@ -1,18 +1,24 @@
...
 
 .. warning:: This means you can't have translations or multiple versions for your documentation.
 
+Having a single version of a documentation project can be considered the better choice
+in cases where there should only always exist one unambiguous copy of your project.
+
+For example, a research project may wish to *only* expose readers to their latest list of publications and research data.
```
Comment (ericholscher) [inline docs/user/single_version.rst:14]:
```suggestion
For example, a research project may wish to *only* expose readers to their latest list of publications and research data.
Similarly, a :abbr:`SaaS (Software as a Service)` application might only ever have one version live. 
```

**Review by ericholscher — APPROVED** [review]

File: docs/user/single_version.rst (hunk 4 above)
Comment (benjaoming) [reply docs/user/single_version.rst:14]: We know that example very well :100:

**Pull request merged**